src/js/features/clear-clutter.js

```diff
@@ -13,4 +13,9 @@ module.exports = function() {
         removeElement('#nav_related_streams');
         $('body').append('<style>#nav_games, #nav_streams, #nav_related_streams, .js-recommended-channels { display: none !important; }</style>');
     }
+
+    // if setting "Hide Last Broadcast Banner" is true
+    if (bttv.settings.get('hideLastBroadcast') === true) {
```

Comment:
> Suggestions for better location?

It should be in its own module.

> And for the CSS in js, I mainly did it because I saw it elsewhere and also to modify the minimal amount of files.

That's because we do have legacy code in some places.. BetterTTV has been around since 2011 so there's straggling old code around, unfortunately.